File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 04/Lab4JAVA/src/main/java/domain/Enrollment.java
package domain;

public class Enrollment {
    Long id;
    Child child;
    Round round;

    /**
     * constructor for enrollment
     *
     * @param child -  child
     * @param round -  round
     */
    public Enrollment(Child child, Round round) {
        this.child = child;
        this.round = round;
    }

    /**
     * getter for child
     *
     * @return - Child object
     */
    public Child getChild() {
        return child;
    }

    /**
     * setter for child
     *
     * @param child - Child object
     */
    public void setChild(Child child) {
        this.child = child;
    }

    /**
     * getter for round
     *
     * @return - Round object
     */
    public Round getRound() {
        return round;
    }

    /**
     * setter for round
     *
     * @param round - Round object
     */
    public void setRound(Round round) {
        this.round = round;
    }

    /**
     * getter methode for id
     *
     * @return - a long
     */
    public Long getId() {
        return id;
    }

    /**
     * setter methode for id
     *
     * @param id - a long
     */
    public void setId(Long id) {
        this.id = id;
    }
}

File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 04/Lab4C#/AthleticsCompetition/domain/Enrollment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AthleticsCompetition.domain
{
    class Enrollment
    {
        /// <summary>
        /// constructor for enrollment
        /// </summary>
        /// <param name="child"> child </param>
        /// <param name="round"> child </param>
        public Enrollment(Child child, Round round)
        {
            this.Child = child;
            this.Round = round;
        }

        /// <summary>
        /// getter and setter for child
        /// </summary>
        public Child Child { get; set; }

        /// <summary>
        /// getter and setter for id 
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// getter and setter for round 
        /// </summary>
        public Round Round { get; set; }

    }
}
